test/src/tests-mParticleUser.js

```diff
@@ -1,46 +1,36 @@
 import Utils from './config/utils';
-import sinon from 'sinon';
 import { urls, apiKey, MPConfig } from './config/constants';
+import fetchMock from 'fetch-mock/esm/client';
 
+const { waitForCondition, fetchMockSuccess } = Utils;
+
+const hasIdentifyReturned = () => {
```

Comment:
Add the cleanup ticket here so we can align this with the other instances of `hasIdentifyReturned`